app/src/androidTest/java/org/mozilla/fenix/ui/SettingsSearchTest.kt

```diff
@@ -348,6 +348,7 @@ class SettingsSearchTest {
         }
     }
 
+    @Ignore("Test failure caused by: https://github.com/mozilla-mobile/fenix/issues/28111")
```

Comment:
We should open a new ticket with details about which tests were disabled and why for QA to know about this and then link that new ticket in the `@Ignore` annotation.